active/0000-ember-forge.md

```diff
@@ -0,0 +1,184 @@
+- Start Date: 2016-04-08
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+The creation of the Ember Forge Foundation to function in the same vein as the [Apache Foundation](http://www.apache.org) and [Dojo Foundation](http://dojofoundation.org), for example. Some examples of their efforts include:
+
+> innovation, by collaborative consensus based processes, an open, pragmatic software license and a desire to create high quality software that leads the way in its field
+- [Apache Foundation](http://www.apache.org)
+
+and
+
+> provide just enough foundation to make great open-source projects succeed and be professional and trustworthy to their users, without bureaucracy or excessive process for projects and their contributors. - [Dojo Foundation](http://dojofoundation.org)
+
+
+Would also provide for the maintenance and management of addons deemed popular or critical by/to the community, that if left unmaintained would be bad for the ecosystem.
+
+
+# Motivation
+
+While developing several applications and addons, and during conversations with other Ember developers, @juwara0 and @notmessenger have identified several parallel motivations for this effort which have been categorized below:
+
+* Evaluation of available addons for use
```

Comment:
I view the Ember Forge Foundation as having larger, broader goals than that of just Ember Addons or Ember Observer by itself. There is no reason that they cannot continue on as it is, with possible additions to align with the Ember Forge Foundation goals, but ultimately it is likely that several different efforts across different tools and spaces will be required to fulfill the goals of the Ember Forge Foundation.